# LiveMpcData Message

std_msgs/Header header

std_msgs/Float32[] x
std_msgs/Float32[] y
std_msgs/Float32[] psi
std_msgs/Float32[] delta
std_msgs/UInt32 qpIterations
std_msgs/UInt64 calculationTime
std_msgs/Float64 cost



# struct LiveMpcData {
#   x @0 :List(Float32);
#   y @1 :List(Float32);
#   psi @2 :List(Float32);
#   delta @3 :List(Float32);
#   qpIterations @4 :UInt32;
#   calculationTime @5 :UInt64;
#   cost @6 :Float64;
# }
